<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>服务器端</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>751</width>
     <height>508</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <widget class="QWidget" name="tab1">
    <attribute name="title">
     <string>欢迎使用</string>
    </attribute>
    <widget class="QLabel" name="labelwel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>361</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cantarell</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
       <stylestrategy>PreferDefault</stylestrategy>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>欢迎使用FedoraMeow点菜系统～～～～</string>
     </property>
    </widget>
    <widget class="QLabel" name="blesslabel">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>360</y>
       <width>301</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>祝您使用愉快～～～～～～～～～</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>150</y>
       <width>431</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="uselabel">
        <property name="text">
         <string>使用指南：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label1">
        <property name="text">
         <string>1.在聊天界面，请先点击“监听”按钮进行连接</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label2">
        <property name="text">
         <string>2.在菜品和厨师资料维护界面可以查看、增加、删除资料</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label3">
        <property name="text">
         <string>3.在餐桌维护界面可以查看餐桌信息和统计报表</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab2">
    <attribute name="title">
     <string>聊天界面</string>
    </attribute>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>351</width>
       <height>381</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>310</y>
       <width>309</width>
       <height>78</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>监听端口</string>
          </property>
          <property name="buddy">
           <cstring>lEPort</cstring>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="lEPort">
            <property name="text">
             <string>3333</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pBBind">
            <property name="text">
             <string>监听</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>发送消息</string>
          </property>
          <property name="buddy">
           <cstring>lEMsg</cstring>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="lEMsg"/>
          </item>
          <item>
           <widget class="QPushButton" name="pBSend">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>发送</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab3">
    <attribute name="title">
     <string>菜品维护界面</string>
    </attribute>
    <widget class="QTableView" name="view">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>371</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="freshButton">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>96</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>刷新数据</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>100</y>
       <width>67</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTextBrowser" name="queBrowser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>281</y>
       <width>371</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>菜品信息窗口</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>菜品查询窗口</string>
     </property>
    </widget>
    <widget class="QPushButton" name="queButton">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>390</y>
       <width>96</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="queEdit">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>350</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>210</y>
       <width>250</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QLineEdit" name="delEdit"/>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="delButton">
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>以编号删除记录</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>320</y>
       <width>157</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>以菜名为关键字查询</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>20</y>
       <width>281</width>
       <height>179</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>增加记录</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="insertButton">
        <property name="text">
         <string>增加</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>编号</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="idEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>菜名</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="nameEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>用时</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="timeEdit"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>价钱</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="welEdit"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab4">
    <attribute name="title">
     <string>厨师资料维护界面</string>
    </attribute>
    <widget class="QTableView" name="view2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>371</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>厨师信息窗口</string>
     </property>
    </widget>
    <widget class="QPushButton" name="freshButton2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>96</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>刷新数据</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>371</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>320</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>以厨师名字为关键字查询</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="queEdit2">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>350</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="queButton2">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>400</y>
       <width>96</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>10</y>
       <width>291</width>
       <height>216</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>增加记录</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="insertButton2">
        <property name="text">
         <string>增加</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>编号</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="idEdit2"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>名字</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="nameEdit2"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>经验</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="expEdit"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>擅长菜式</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="goodEdit"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>好评度</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="welEdit2"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>240</y>
       <width>250</width>
       <height>60</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>以编号删除</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="delEdit2"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="delButton2">
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab5">
    <attribute name="title">
     <string>餐桌维护界面</string>
    </attribute>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>餐桌使用情况：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="freshButton3">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>10</y>
       <width>96</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>刷新数据</string>
     </property>
    </widget>
    <widget class="QTableView" name="view3">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>41</y>
       <width>411</width>
       <height>391</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser2">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>150</y>
       <width>281</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="todayButton">
     <property name="geometry">
      <rect>
       <x>585</x>
       <y>110</y>
       <width>141</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>查看今日情况</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>newConn()</slot>
  <slot>onData()</slot>
  <slot>on_pBBind_clicked()</slot>
  <slot>on_pBSend_clicked()</slot>
 </slots>
</ui>
